app/assets/stylesheets/dark-mode.css

```
@@ -1,7 +1,7 @@
 :root {
   --dark-bg-color: #121212;
   --dark-bg-accent: #1a1a1a;
-  --dark-font-color: #b3b3b3;
+  --dark-font-color: #cac7c7;
```

Hey @ArudoPE thanks for doing this. However, I don't think that color fits. It looks off when paired with headings.  

Playing around with it, white with a slightly reduced opacity seems to work better:
```suggestion
  --dark-font-color: rgba(255, 255, 255, 0.87);
```

---

Yes, actually, that would be even better. I tried to not get too close to white, but that's a nice suggestion.